src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2454,6 +2460,285 @@ static bool forwardsPending(UpdateContextResponse* upcrsP)
   return false;
 }
 
+/* ****************************************************************************
+*
+* updateEntity -
+*/
+static void updateEntity
+(
+  const BSONObj&                  r,
+  const std::string&              action,
+  const std::string&              tenant,
+  const std::vector<std::string>& servicePathV,
+  const std::string&              xauthToken,
+  ContextElement*                 ceP,
+  UpdateContextResponse*          responseP,
+  bool*                           attributeAlreadyExistsError,
+  std::string*                    attributeAlreadyExistsList
+)
+{
+  // Used to accumulate error response information
+  *attributeAlreadyExistsError = false;
+  *attributeAlreadyExistsList  = "[ ";
+
+  EntityId*          enP               = &ceP->entityId;
+  const std::string  idString          = "_id." ENT_ENTITY_ID;
+  const std::string  typeString        = "_id." ENT_ENTITY_TYPE;
+  const std::string  servicePathString = "_id." ENT_SERVICE_PATH;
+
+  BSONElement idField = getField(r, "_id");
+
+  std::string entityId    = getStringField(idField.embeddedObject(), ENT_ENTITY_ID);
+  std::string entityType  = getStringField(idField.embeddedObject(), ENT_ENTITY_TYPE);
+  std::string entitySPath = getStringField(idField.embeddedObject(), ENT_SERVICE_PATH);
+
```

Comment:
This block of three variables is aligned... Do you mean to align all from EntityId* to line 2494?